ansible-ipi-install/roles/node-prep/tasks/10_validation.yml

```diff
@@ -22,6 +22,30 @@
   - always
   - validation
 
+- name: Ensure Python cryptography libraries are installed
+  python_requirements_info:
+    dependencies:
+    - cryptography>=1.2.3
+  delegate_to: localhost
+  register: _py_crypto
+  ignore_errors: yes
+
+- name: Ensure Python PyOpenSSL libraries are installed
+  python_requirements_info:
+    dependencies:
+    - PyOpenSSL>=0.6
+  delegate_to: localhost
+  register: _py_pyopenssl
+  ignore_errors: yes
+
+- name: "Fail on missing required cryptography libraries cryptography OR PyOpenSSL"
+  fail:
+    msg: "Required cryptography libraries are missing cryptography OR PyOpenSSL"
+  when: (_py_crypto.not_found != []) and
+        (_py_pyopenssl.not_found != [] )
+  tags:
+    - create_registry
```

Comment:
you will want the tags: validation in here as well

Reply:
But we only want to check if libraries are installed in the case we want to create a registry, and ignore it for any other case, isn't it?